src/vaadin-combo-box.html

```diff
@@ -397,9 +397,8 @@
           // We need to set this listener for "this.inputElement"
           // instead of just "this", otherwise keyboard navigation behaviour
           // breaks a bit on Safari and some related tests fail.
```

Comment:
Need to remove this comment in case the change below is fine (tests seem to pass fine on Safari)